CHANGELOG.md

```diff
@@ -26,7 +26,7 @@ Version 4.6.0 *(2017-10-30)*
 
  * New: Lint checks have been ported to UAST, their stability improved, and quick-fix suggestions added. They require Android Gradle Plugin 3.0 or newer to run.
  * New: Added nullability annotations for Kotlin users.
- * Fix: Tag length truncation no longer occurs on API 24 or newer as the system no longer has a length restriction.
+ * Fix: Tag length truncation no longer occurs on API ~24~ 26 or newer as the system no longer has a length restriction.
```

Comment:
```suggestion
 * Fix: Tag length truncation no longer occurs on API 24 or newer as the system no longer has a length restriction.
```
The change log is accurate with the library behavior at this version.

Reply:
True. I was more focused on someone using that as canonical proof that tag length want an issue starting from API 24.

Maybe a note pointing out that it's not it's more appropriate.